src/User.cpp

```diff
@@ -25,6 +25,17 @@ std::string User::getAllInfo() const
     ss<<std::endl;
     return ss.str();
 }
+std::string User::getAllInfoToFile() const
+{
+    std::stringstream ss;
+    ss<<std::left;
+    ss<<std::setw(1) << name_ << std::endl;
+    ss<<std::setw(1) << nick_ << std::endl;
+    ss<<std::setw(1) << group_ << std::endl;
+    ss<<std::setw(1) << gitHub_ << std::endl;
+    ss<<std::setw(1) << firecode_ << std::endl;
+    return ss.str();
+}
```

Comment:
hmm... isn't this almost the same as getAllInfo()? Maybe one of those functions is redundant? It's code duplication.